Gems/Atom/Asset/ImageProcessingAtom/Assets/Config/ImageBuilder.settings

```diff
@@ -0,0 +1,150 @@
+            "_spec": [ "Reflectance" ],
+            "_specular": [ "Reflectance" ],
+            "_metallic": [ "Reflectance" ],
+            "_refl": [ "Reflectance" ],
+            "_ref": [ "Reflectance" ],
+            "_rf": [ "Reflectance" ],
+            "_gloss": [ "Reflectance" ],
+            "_g": [ "Reflectance" ],
+            "_f0": [ "Reflectance" ],
+            "_specf0": [ "Reflectance" ],
+            "_metal": [ "Reflectance" ],
+            "_mtl": [ "Reflectance" ],
+            "_m": [ "Reflectance" ],
+            "_mt": [ "Reflectance" ],
+            "_metalness": [ "Reflectance" ],
+            "_rough": [ "Reflectance" ],
+            "_roughness": [ "Reflectance" ],
+            // opacity
+            "_sss": [ "Opacity" ],
+            "_trans": [ "Opacity" ],
+            "_opac": [ "Opacity" ],
+            "_opacity": [ "Opacity" ],
+            "_o": [ "Opacity" ],
+            "_op": [ "Opacity" ],
+            "_mask": [ "Opacity", "Greyscale" ],
+            "_msk": [ "Opacity" ],
+            "_blend": [ "Opacity" ],
+            // AO
+            "_ao": [ "AmbientOcclusion" ],
+            "_ambocc": [ "AmbientOcclusion" ],
+            "_amb": [ "AmbientOcclusion" ],
+            "_ambientocclusion": [ "AmbientOcclusion" ],
+            // emissive
+            "_emissive": [ "Emissive" ],
+            "_e": [ "Emissive" ],
+            "_glow": [ "Emissive" ],
+            "_em": [ "Emissive" ],
+            "_emit": [ "Emissive" ],
+            // displacement
+            "_displ": [ "Emissive" ],
+            "_disp": [ "Emissive" ],
+            "_dsp": [ "Emissive" ],
+            "_d": [ "Emissive" ],
+            "_dm": [ "Emissive" ],
+            "_displacement": [ "Emissive" ],
+            "_height": [ "Emissive" ],
+            "_hm": [ "Emissive" ],
+            "_ht": [ "Emissive" ],
+            "_h": [ "Emissive" ],
+            // cubemap
+            "_ibldiffusecm": [ "IBLDiffuse" ],
+            "_iblskyboxcm": [ "IBLSkybox" ],
+            "_iblspecularcm": [ "IBLSpecular" ],
+            "_iblspecularcm128": [ "IBLSpecularLow" ],
+            "_iblspecularcm256": [ "IBLSpecular" ],
+            "_iblspecularcm512": [ "IBLSpecularHigh" ],
+            "_iblspecularcm1024": [ "IBLSpecularVeryHigh" ],
+            "_skyboxcm": [ "Skybox" ],
+            "_ccm": [ "ConvolvedCubemap" ],
+            "_convolvedcubemap": [ "ConvolvedCubemap" ],
+            "_iblglobalcm": [ "IBLGlobal" ],
+            "_cubemap": [ "IBLGlobal" ],
+            "_cm": [ "IBLGlobal" ],
+            // lut
+            "_lut": [ "LUT_RG8" ],
+            "_lutr32f": [ "LUT_R32F" ],
+            "_lutrgba16": [ "LUT_RGBA16" ],
+            "_lutrg32f": [ "LUT_RG32F" ],
+            "_lutrgba32f": [ "LUT_RGBA32F" ],
+            // layer mask
+            "_layers": [ "LayerMask" ],
+            "_rgbmask": [ "LayerMask" ],
+            // decal
+            "_decal": [ "Decal_AlbedoWithOpacity" ],
```

Comment:
This case is a bit of different. it's reference to a preset with this name. But it's not necessary to the xxx.preset file in this folder. because it can be referencing the xxx.preset in project folder. I think without the .preset could avoid the expectation that the file in the same folder would be the one. 
I will add a document to explain all the different type of setting files.